packages/js-client-rest/tests/integration/qdrant-client.test.ts

```diff
@@ -7,6 +8,9 @@ describe('QdrantClient', () => {
     const DIM = 100;
     const client = new QdrantClient();
     const collectionName = 'test_collection';
+    const bigInt = BigInt(String(Number.MAX_SAFE_INTEGER + 2)) as unknown as number;
```

Comment:
If we try to convert a BigInt to a number directly, it may result in loss of precision for large BigInt values. That's why TypeScript requires you to use the as unknown as number syntax to explicitly acknowledge that you are performing an unsafe type conversion.

This is known as double assertion.